src/plugins/guided_onboarding/public/components/guide_panel.styles.ts

```diff
@@ -16,16 +19,24 @@ import { css } from '@emotion/react';
  * In order to leverage a11y features, we are using the EuiFlyout and applying customizations
  * See https://github.com/elastic/eui/issues/6241 for more details
  */
-export const getGuidePanelStyles = (euiTheme: EuiThemeComputed) => ({
+export const getGuidePanelStyles = ({
+  euiTheme,
+  isDarkTheme,
+}: {
+  euiTheme: EuiThemeComputed;
+  isDarkTheme: boolean;
+}) => ({
   setupButton: css`
     margin-right: ${euiTheme.size.m};
   `,
   flyoutOverrides: {
+    flyoutHeader: css`
+      background: url(${panelBgTop}) top right no-repeat;
```

Comment:
I think Cindy also [linked](https://user-images.githubusercontent.com/11224465/214531956-1604244b-404c-483f-a7df-562d1f77115a.svg) an SVG file for the top panel in the dark theme.